extract-docker-tag/main.go

```diff
@@ -42,19 +42,22 @@ func main() {
 
 	action.SetOutput("owner", result.owner)
 	action.SetOutput("name", result.name)
-	action.SetOutput("tag", result.tag)
+	action.SetOutput("tag", strings.Join(result.tags, ","))
```

Comment:
```suggestion
	action.SetOutput("tags", strings.Join(result.tags, ","))
```

We also should update action.yml file

Reply:
Missed it, fixed it now.